docs/guidance/mobilenet-rayservice.md

```diff
@@ -0,0 +1,42 @@
+# RayService: MobileNet example
+
+Both Python scripts for Ray Serve and sending requests are in the repository [kevin85421/ray-serve-examples](https://github.com/kevin85421/ray-serve-examples).
+
+## Step 1: Create a Kubernetes cluster with Kind.
+
+```sh
+kind create cluster --image=kindest/node:v1.23.0
+```
+
+## Step 2: Install KubeRay operator
+
+Follow this [document](../../helm-chart/kuberay-operator/README.md) to install the latest stable KubeRay operator via Helm repository.
+
+## Step 3: Install a RayService
+
+```sh
+# path: ray-operator/config/samples/
+kubectl apply -f ray-service.mobilenet.yaml
+```
+
+* The [mobilenet.py](https://github.com/kevin85421/ray-serve-examples/blob/main/mobilenet.py) requires `tensorflow` as a dependency. Hence, the YAML file uses `rayproject/ray-ml:2.5.0` instead of `rayproject/ray:2.5.0`.
+* `python-multipart` is required for the request parsing function `starlette.requests.form()`, so the YAML file includes `python-multipart` in the runtime environment.
+
+## Step 4: Forward the port of Serve
+
+```sh
+kubectl port-forward svc/rayservice-mobilenet-serve-svc 8000
+```
+
+Note that the Serve service will be created after the Serve applications are ready and running. This process may take some time.
+
+## Step 5: Send a request to the ImageClassifier
+
+* Step 5.1: Prepare an image file (e.g. golden.png)
+* Step 5.2: Update `image_path` in [mobilenet_req.py](https://github.com/kevin85421/ray-serve-examples/blob/main/mobilenet_req.py
+)
+* Step 5.3: Send a request to the ImageClassifier.
+  ```sh
+  python3 mobilenet_req.py
```

Comment:
It's better to use `python` instead of `python3` here to maintain consistency with the rest of the docs.

Reply:
Fixed by https://github.com/ray-project/kuberay/pull/1175/commits/c9431d28fd2dd135fd1c497afc4c2a3e6ddfe4e2